edgelet/iotedged/src/lib.rs

```diff
@@ -881,6 +889,8 @@ where
     let label = "mgmt".to_string();
     let url = settings.listen().management_uri().clone();
 
+    let cert_manager = cert_manager.clone();
```

Comment:
If you want to own it here, why clone a borrow instead of taking it by value in the first place?

Same in `start_workload`